### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

**Review by IhorVoskoboinikov — APPROVED** [review]
As for me your code is well) I run your code and PyCharm say to me that have to use this annotation: Optional[Skill] (import from typing) if you return None from function.

File: main.py [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def get_race(race_info: dict) -> Race:
+    result = Race.objects.get_or_create(
+        name=race_info["name"],
+        description=race_info["description"]
+    )[0]
```
Comment (YuriiKindrat) [inline main.py:11]: result is bad name for variable, also it is better to use underscore here, instead of indexes ))

Comment (YuriiKindrat) [reply main.py:11]: same is for similar cases

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
 from db.models import Race, Skill, Player, Guild
 
 
+def get_race(race_info: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=race_info["name"],
+        description=race_info["description"]
+    )
+    for skill_info in race_info["skills"]:
+        _ = get_skill(skill_info, race)
```
Comment (Oleg-s-s) [inline main.py:13]: Why you create a var here?

File: main.py [hunk 3]
```diff
@@ -1,10 +1,53 @@
...
+    )
+    return skill
+
+
 def main() -> None:
-    pass
+    with open("players.json", "r") as data_source:
+        players = json.load(data_source)
+    for player_name, player_data in players.items():
+        _ = Player.objects.get_or_create(
```
Comment (Oleg-s-s) [inline main.py:42]: And here

**Review by Oleg-s-s — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (thorium-zero) [reply main.py:13]: @Oleg-s-s Discards are placeholder variables that are intentionally unused in application code. Discards make the intent of your code clear. A discard indicates that our code never uses the variable. They enhance its readability and maintainability.

File: main.py (hunk 3 above)
Comment (thorium-zero) [reply main.py:42]: same here

**Pull request closed without merging**